st2common/st2common/transport/announcement.py

```diff
@@ -20,16 +20,20 @@
 from st2common.constants.trace import TRACE_CONTEXT
 from st2common.models.api.trace import TraceContext
 from st2common.transport import publishers
-from st2common.transport import utils as transport_utils
 
 LOG = logging.getLogger(__name__)
 
 # Exchange for Announcements
 ANNOUNCEMENT_XCHG = Exchange('st2.announcement', type='topic')
 
+__all__ = [
+    'AnnouncementPublisher',
+    'AnnouncementDispatcher'
+]
+
 
 class AnnouncementPublisher(object):
-    def __init__(self, urls):
+    def __init__(self, urls=None):
```

Comment:
Connection retrieval and URL handling is now centralized in a single place so there is really no need for this ``url`` argument here anymore.

To make the code nicer, I could simply remove it from all those classes.